SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -168,10 +185,10 @@ bool CreatePlaceholders(Worker_SnapshotOutputStream* OutputStream)
 			ComponentWriteAcl.Add(SpatialConstants::PERSISTENCE_COMPONENT_ID, SpatialConstants::UnrealServerPermission);
 			ComponentWriteAcl.Add(SpatialConstants::ENTITY_ACL_COMPONENT_ID, SpatialConstants::UnrealServerPermission);
 
-			Components.Add(Position(PlaceholderPosition).CreatePositionData());
-			Components.Add(Metadata(TEXT("Placeholder")).CreateMetadataData());
-			Components.Add(Persistence().CreatePersistenceData());
-			Components.Add(EntityAcl(SpatialConstants::UnrealServerPermission, ComponentWriteAcl).CreateEntityAclData());
+			Components.Add(SpatialGDK::Position(PlaceholderPosition).CreatePositionData());
+			Components.Add(SpatialGDK::Metadata(TEXT("Placeholder")).CreateMetadataData());
+			Components.Add(SpatialGDK::Persistence().CreatePersistenceData());
```

Comment:
why is hydra introducing these namespace changes, I though these were made in master?

Reply:
Hydra was introducing improbable. but since there a using these are not needed, will update now